### DISCUSSION TITLE
Please give us any infomation or feedback on Monthly meetings 2022-11

### DISCUSSION THREAD

**Discussion by yuiseki** [UNopenGIS/7#24] [Ideas]
To: @hfu 

I see you and others had Monthly meeting with the UN Open GIS Initiative.

- Related: #11

I believe this meeting is important for us to be useful to the UN.

So I would love to know the following! : 
- How was our activity reported and evaluated?
- What were the reports of the other working groups?
- What are we expected to do next?
  - How do we answer that expectation?

I hope you'll take advantage of the Announcements category on GitHub Discussions for the above!

**Comment by hfu** [comment]
Thanks! I created https://github.com/UNopenGIS/7/discussions/25 as suggested.
